Scientific publications where Cell-ACDC was used
================================================

In the following publications authors used Cell-ACDC to analyse microscopy data:

.. raw:: html

    <ol>
      <li>Dengler, L. et al. <i>When mitochondria fall apart: Unbalanced mitochondrial segregation triggers loss of mtDNA in the absence of mitochondrial fusion</i>. <b>bioRxiv</b> (2025) DOI: <a href="https://doi.org/10.1101/2025.05.13.653688">10.1101/2025.05.13.653688</a>.</li>
      <li>Seshadri, A. et al. <i>Exonuclease action of replicative polymerase gamma drives damage-induced mitochondrial DNA clearance</i>. <b>EMBO Reports</b> (2025) DOI: <a href="https://doi.org/10.1038/s44319-025-00380-1">10.1038/s44319-025-00380-1</a>.</li><br>
      <li>Al-Refaie, N. et al. <i>Fasting shapes chromatin architecture through an mTOR/RNA Pol I axis</i>. <b>Nat. Cell Biol.</b> 1–15 (2024) DOI: <a href="https://doi.org/10.1038/s41556-024-01512-w">10.1038/s41556-024-01512-w</a>.</li><br>
      <li>Lanz, M. C. et al. <i>Genome dilution by cell growth drives starvation-like proteome remodeling in mammalian and yeast cells</i>. <b>Nat. Struct. Mol. Biol.</b> (2024) DOI: <a href="https://doi.org/10.1038/s41594-024-01353-z">10.1038/s41594-024-01353-z</a>.</li><br>
      <li>Xiao, J., Turner, J. J., Kõivomägi, M. & Skotheim, J. M. <i>Whi5 hypo- and hyper-phosphorylation dynamics control cell-cycle entry and progression</i>. <b>Curr. Biol.</b> 34, 2434-2447.e5 (2024).DOI: <a href="https://doi.org/10.1016/j.cub.2024.04.052">10.1016/j.cub.2024.04.052</a>.</li><br>
      <li>Vitacolonna, M. et al. <i>A multiparametric analysis including single-cell and subcellular feature assessment reveals differential behavior of spheroid cultures on distinct ultra-low attachment plate types</i>. <b>Front. Bioeng. Biotechnol.</b> 12, (2024) DOI: <a href="https://doi.org/10.3389/fbioe.2024.1422235">10.3389/fbioe.2024.1422235</a>.</li><br>
      <li>Roussou, R. et al. <i>Real-time assessment of mitochondrial DNA heteroplasmy dynamics at the single-cell level</i>. <b>EMBO J.</b> 43, 5340–5359 (2024) DOI: <a href="https://doi.org/10.1038/s44318-024-00183-5">10.1038/s44318-024-00183-5</a>.</li><br>
      <li>Padovani, F. et al. <i>SpotMAX: a generalist framework for multi-dimensional automatic spot detection and quantification</i>. <b>bioRxiv</b> (2024) DOI: <a href="https://doi.org/10.1101/2024.10.22.619610">10.1101/2024.10.22.619610</a>.</li><br>
      <li>Kukhtevich, I. et al. <i>The origin of septin ring size control in budding yeast</i>. <b>bioRxiv</b> (2024) DOI: <a href="https://doi.org/10.1101/2024.07.30.605628">10.1101/2024.07.30.605628</a>.</li><br>
      <li>Chatzitheodoridou, D., Bureik, D., Padovani, F., Nadimpalli, K. V. & Schmoller, K. M. <i>Decoupled transcript and protein concentrations ensure histone homeostasis in different nutrients</i>. <b>EMBO J.</b> 43, 5141–5168 (2024) DOI: <a href="https://doi.org/10.1038/s44318-024-00227-w">10.1038/s44318-024-00227-w</a>.</li><br>
      <li>Chadha, Y., Kukhtevich, I. V., Padovani, F., Schneider, R. & Schmoller, K. M. <i>Single-cell imaging reveals a key role of Bck2 in budding yeast cell size adaptation to nutrient challenges</i>. <b>bioRxiv</b> (2024) DOI: <a href="https://doi.org/10.1101/2024.10.04.616606">10.1101/2024.10.04.616606</a>.</li><br>
      <li>Seel, A. et al. <i>Regulation with cell size ensures mitochondrial DNA homeostasis during cell growth</i>. <b>Nat. Struct. Mol. Biol.</b> 30, 1549–1560 (2023) DOI: <a href="https://doi.org/10.1038/s41594-023-01091-8">10.1038/s41594-023-01091-8</a>.</li><br>
      <li>Piñeiro López, C., Rodrigues Neves, A. R., Čavka, I., Gros, O. J. & Köhler, S. <i>Segmentation of C. elegans germline nuclei</i>. <b>MicroPubl Biol.</b> (2023) DOI: <a href="https://doi.org/10.17912/MICROPUB.BIOLOGY.001062">10.17912/MICROPUB.BIOLOGY.001062</a>.</li><br>
      <li>Freitag, M. et al. <i>Single-molecule experiments reveal the elbow as an essential folding guide in SMC coiled-coil arms</i>. <b>Biophys. J.</b> 121, 4702–4713 (2022) DOI: <a href="https://doi.org/10.1016/j.bpj.2022.10.017">10.1016/j.bpj.2022.10.017</a>.</li><br>
      <li>Kukhtevich, I. V. et al. <i>Quantitative RNA imaging in single live cells reveals age-dependent asymmetric inheritance</i>. <b>Cell Rep.</b> 41, (2022) DOI: <a href="https://doi.org/10.1016/j.celrep.2022.111656">10.1016/j.celrep.2022.111656</a>.</li><br>
      <li>Padovani, F., Mairhörmann, B., Falter-Braun, P., Lengefeld, J. & Schmoller, K. M. <i>Segmentation, tracking and cell cycle analysis of live-cell imaging data with Cell-ACDC</i>. <b>BMC Biol.</b> 20, 174 (2022) DOI: <a href="https://doi.org/10.1186/s12915-022-01372-6">10.1186/s12915-022-01372-6</a>.</li><br>
      <li>Schuh, L. et al. <i>Altered expression response upon repeated gene repression in single yeast cells</i>. <b>PLOS Comput. Biol.</b> 18, e1010640 (2022) DOI: <a href="https://doi.org/10.1371/journal.pcbi.1010640">10.1371/journal.pcbi.1010640</a>.</li>
    </ol>